rules/group.go

```diff
@@ -244,7 +244,8 @@ func (g *Group) run(ctx context.Context) {
 			select {
 			case <-g.managerDone:
 			case <-time.After(2 * g.interval):
-				g.cleanupStaleSeries(ctx, now)
+				hints := &storage.AppendHints{DiscardOutOfOrder: true}
```

Comment:
This should probably be some default hints config instead of instantiated every interval.